interfaces/cython/cantera/onedim.pyx

```diff
@@ -507,6 +514,28 @@ cdef class IonFlow(_FlowBase):
                 (<CxxIonFlow*>self.flow).fixElectricPotential()
 
 
+cdef class IonFreeFlow(IonFlow):
+    """
+    A domain of free flow with ionized gas.
+    """
+    def __cinit__(self, _SolutionBase thermo, *args, **kwargs):
+        gas = getIdealGasPhase(thermo)
+        self.flow = <CxxStFlow*>(new CxxIonFlow(gas, thermo.n_species, 2))
+        self.set_freeFlow()
+        self.set_viscosityFlag(False)
```

Comment:
I think the Python wrapper classes should mirror the C++ classes for the "flow" domain, i.e. there should only be one that wraps `IonFlow` and one that wraps `StFlow` (which arguably should be called `IdealGasFlow`), and then the classes for the entire flame (e.g. `BurnerIonFlame`) can call the method to set axisymmetric vs free flow.